distributions/distributions.yaml

```diff
@@ -1169,6 +1169,30 @@ sources:
       binaries:
         - gh
 
+  git-bug:
+    description: Distributed, offline-first bug tracker embedded in git, with bridges
```

Comment:
can you also add

`url: https://github.com/MichaelMure/git-bug/` 

please ?

Reply:
Added.